### PR TITLE
[DUOS-665][risk=low] Remove use restriction info from Election

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
https://broadinstitute.atlassian.net/browse/DUOS-665
Requires: https://github.com/DataBiosphere/consent/pull/838

## Changes
* Remove `useRestriction` from Election
* Remove `translatedUseRestriction` from Election
* Where those values were used, get them from the consent instead

----

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing
- [ ] I've updated Swagger to reflect any API changes

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/java/org/broadinstitute/consent/http/models/ElectionReview.java [hunk 1]
```diff
@@ -79,9 +79,6 @@ public void setAssociatedConsent(Consent consent, Election consentElection) {
         this.associatedConsent = new Document();
         this.associatedConsent.put("electionId", consentElection != null ? consentElection.getElectionId() : null);
         this.associatedConsent.put("consentId", consent.getConsentId());
-        this.associatedConsent.put("useRestriction", consentElection != null ? consentElection.getUseRestriction() : consent.getUseRestriction());
-        this.associatedConsent.put("translatedUseRestriction", consentElection != null ? consentElection.getTranslatedUseRestriction() : consent.getTranslatedUseRestriction());
-        this.associatedConsent.put("dulName", consentElection != null ? consentElection.getDulName() : consent.getDulName());
```
Comment (lindojong) [inline src/main/java/org/broadinstitute/consent/http/models/ElectionReview.java]: Did you mean to remove dulName here?

Comment (rushtong) [reply src/main/java/org/broadinstitute/consent/http/models/ElectionReview.java]: Yes, it is unused in the UI

Comment (rushtong) [reply src/main/java/org/broadinstitute/consent/http/models/ElectionReview.java]: This needs to be re-based ... I removed these fields in https://github.com/DataBiosphere/consent/pull/838

**Review by JVThomas — COMMENTED** [review]
On ElectionDAO, line 116, there's a segment of code as such:

`@SqlQuery(" SELECT DISTINCT e.electionid, e.datasetid, v.vote finalvote, e.status, e.createdate, e.referenceid, " +	    @SqlQuery(" SELECT DISTINCT e.electionid, e.datasetid, v.vote finalvote, e.status, e.createdate, e.referenceid, " +
            "   e.userestriction, e.translateduserestriction, v.rationale finalrationale, v.createdate finalvotedate, " +	            "   e.userestriction, e.translateduserestriction`

Should "userestriction and "translateduserestriction" be included here?

**Comment by rushtong** [comment]
@JVThomas Yes, looks like I missed those lines, thank you!

**Comment by rushtong** [comment]
@JVThomas Updated, thank you!

**Review by lindojong — APPROVED** [review]

**Pull request merged**